cogs/highlights.py

```diff
@@ -69,11 +69,21 @@ async def _send_highlights(
                     continue
         await con.commit()
 
+    def _get_guild_id(self, server: discord.Guild | None) -> int:
```

Comment:
This method doesn't use the `self` argument so it should be a function instead of a method. Actually it doesn't do enough to justify creating a function either, this should really just be in-lined.

Reply:
Fixed: inlined